hummingbot/core/data_type/common.py

```diff
@@ -7,6 +7,10 @@ class OrderType(Enum):
     MARKET = 1
     LIMIT = 2
     LIMIT_MAKER = 3
+    # New order types
+    STOP_LOSS = 4
+    TAKE_PROFIT = 5
+    TRAILING_STOP = 6
 
     def is_limit_type(self):
```

Comment:
On Kraken, TAKE_PROFIT has a different entry form and it executes as a MARKET order when the % or the price difference is matched, same for the other. Kraken does have limit equivalent to these. I will commit my modifications of the Kraken connector then - It will take a bit longer, I need to clean-up and ad tests in this case